### PR TITLE
MeshLambertMaterial: add support for LightProbes

### PR DESCRIPTION
**Pull request by WestLangley**

### CODE REVIEW

File: src/renderers/shaders/ShaderLib/meshlambert_frag.glsl.js [hunk 1]
```diff
@@ -52,7 +52,6 @@ void main() {
 	#include <emissivemap_fragment>
 
 	// accumulation
```
Comment (mrdoob) [inline src/renderers/shaders/ShaderLib/meshlambert_frag.glsl.js:54]: Minor: Do we still need this comment?

Comment (WestLangley) [reply src/renderers/shaders/ShaderLib/meshlambert_frag.glsl.js:54]: @mrdoob It refers to lines 54 to 81.

Comment (mrdoob) [reply src/renderers/shaders/ShaderLib/meshlambert_frag.glsl.js:54]: Ah, I see.

**Comment by mrdoob** [comment]
Thanks!

**Pull request merged**